Sources/FoundationEssentials/ProcessInfo/ProcessInfo.swift

```diff
@@ -140,9 +144,28 @@ final class _ProcessInfo: Sendable {
             return username
         }
         return ""
-#else
-        // TODO: Windows
+#elseif os(WASI)
+        // WASI does not have user concept
         return ""
+#elseif os(Windows)
+        return withUnsafeTemporaryAllocation(of: wchar_t.self, capacity: 1040) { usernameBuffer in
```

Comment:
Is this meant to be the user that the process is running as or whatever user is logged in? I wonder if we should query the information from the process.

Reply:
I believe it should be the "current user" (i.e. whichever user is logged in) based on what I've heard from @iCharlesHu - this code was mostly just copied from swift-corelibs-foundation as-is so I think it should behave the same as what SCF currently does